.github/workflows/ci.yml

```diff
@@ -19,25 +19,31 @@ jobs:
   test:
     name: Run pipeline with test data
     # Only run on push if this is the nf-core dev branch (merged PRs)
-    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'fellen31/skierfe') }}"
+    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'genomic-medicine-sweden/nallo') }}"
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        parameters:
+          - ""
```

Comment:
A test with no parameters?

Reply:
Yes, so to run the -profile test with no extra arguments.